scivision/catalog/data/datasources.json

```
@@ -276,6 +276,27 @@
       "description": "Plate 20585 - Phenotypic profiling attempts to summarize multiparametric, feature-based analysis of cellular phenotypes of each sample so that similarities between profiles reflect similarities between samples. This image set provides a basis for testing image-based profiling methods wrt. to their ability to distinguish the effects of small molecules. Images The images are of U2OS cells treated with each of 1600 known bioactive compounds and labeled with six labels that characterize seven organelles (the .cell-painting. assay). This pilot experiment consists of 20 plates 5 of which are available. Each plate has 384 wells and each well has 9 fields of view for a total of 17,280 fields of view. Each field was imaged in five channels (detection wavelengths), and each channel is stored as a separate, grayscale image file, so there are 86,400 image files in 16-bit TIFF format.",
       "url": "https://github.com/Tonks684/phenotypic_profiling/blob/main/data.yml",
       "format": ".tif"
+    },
+    {
+      "tasks": [
+        "classification"
+      ],
+      "labels_provided": true,
+      "domains": [
+        "history",
+        "urban-planning"
+      ],
+      "institution": [
+        "The Alan Turing Institute"
+      ],
+      "tags": [
+        "2D",
+        "humanities"
+      ],
+      "name": " MapReader_Data_SIGSPATIAL_2022 ",
+      "description": "Here we share gold standard annotations and outputs from early experiments using MapReader. MapReader creates datasets for humanities research using historical map scans and metadata as inputs.  Using maps provided by the National Library of Scotland, these annotations and outputs reflect labeling tasks relevant to historical research on the Living with Machines project.  Data shared here is derived from maps printed in nineteenth-century Britain by the Ordnance Survey, Britain's state mapping agency. These maps cover England, Wales, and Scotland from 1888 to 1913.",
+      "url": "https://raw.githubusercontent.com/alan-turing-institute/mapreader-scivision/main/.scivision/data.yml?token=GHSAT0AAAAAAB5AKAF3DMD6JR57GJTCPX7MY63VFSQ",
```

```suggestion
      "url": "https://github.com/alan-turing-institute/mapreader-plant-scivision",
```
The repo is public, but there shouldn't be a token in the url.
Since the config is at `.scivision/data.yml` in the repo, this simple url should work ^